.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 
 # review when someone opens a pull request.
 
-- @Fosol @jdtoombs @kylgarmor @CelloLiu
+- @Fosol @jdtoombs @kylgarmor @CelloLiu @fbarreta @mihailistov
```

Comment:
@fbarreta @mihailistov you should now receive emails when a PR is created.  These emails go to your github account.